metricbeat/Makefile

```diff
@@ -1,5 +1,5 @@
 # Name can be overwritten, as Metricbeat is also a library
-BEAT_NAME?=metricbeat
+export BEAT_NAME?=metricbeat
```

Comment:
Why is this change needed?

Reply:
Added it so the Magefile could pick it out with `getEnv`, but unnecessary with the helpers.